pandas/tests/groupby/test_aggregate.py

```diff
@@ -302,6 +304,24 @@ def test_aggregate_api_consistency(self):
                                                     ['D', 'C']])
         assert_frame_equal(result, expected, check_like=True)
 
+    def test_agg_dict_renaming_deprecation(self):
+        # 15931
+        df = pd.DataFrame({'A': [1, 1, 1, 2, 2],
+                           'B': range(5),
+                           'C': range(5)})
+
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False) as w:
+            df.groupby('A').agg({'B': {'foo': ['sum', 'max']},
+                                 'C': {'bar': ['count', 'min']}})
+            assert "using a dict with renaming" in str(w[0].message)
+
```

Comment:
It is tested in other cases as well, but since this is a test specifically for the deprs, maybe also add the case of `df.groupby('A')[['B', 'C']].agg({'ma': 'max'})` (then you have the different 'cases' that raise deprecation here)